CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## 5.0.3
+  - Fix bug where codec was not flushed when client disconnected
+  - Restore INFO logging statement on startup
   - Update gemspec summary
```

Comment:
please include a mention to https://github.com/logstash-plugins/logstash-input-tcp/pull/94 as well